### DISCUSSION TITLE
How to configure proxy for SQS in Spring Cloud AWS 3

### DISCUSSION THREAD

**Discussion by ghughal** [awspring/spring-cloud-aws#1027] [General]
With 2.4.x version, we were using `ClientConfiguration` to configure proxy (and other properties like max retry, max connection etc.) but it's not clear how to do that programmatically with 3.1.x version and it was not clear from documentation. Thank you.

**Comment by ifeLawal** [comment]
Is there anyone working on this / does anyone know if this discussion is being viewed by the awsspring team? My team and I are also looking for guidance on this

**Comment by maciejwalkowiak** [comment]
While far from perfect, you can customize each AWS client like explained in docs: https://docs.awspring.io/spring-cloud-aws/docs/3.2.0/reference/html/index.html#customizing-aws-clients
